### DISCUSSION TITLE
How to add a (upload datasets) function and use the uploaded datasets?

### DISCUSSION THREAD

**Discussion by botsp** [insightsengineering/teal.gallery#88] [Q&A] [answered]
Wish for your suggestions

**Comment by lcd2yyz** [comment, marked as answer]
Hi @botsp thanks for your question!
This is not possible at the moment. But good news is that we are currently making some changes to the way teal ingest/passes data through the framework, which should open up the possibility for such new feature in the future. Please stay tuned.

**Reply by botsp** [reply]
yeah, Thank you The interactivity option would reduce more codes. really appreciate your work!

**Reply by Mia-data** [reply]
The possibility of uploading datasets would be very helpful for my project too! I'm looking forward to it!
